snmp/datadog_checks/snmp/data/profiles/_cisco-generic.yaml

```diff
@@ -8,6 +8,24 @@ extends:
   - _generic-bgp4.yaml
   - _generic-ip.yaml
 
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "cisco"
+      version:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          extract_value: '\sVersion\s+([a-zA-Z0-9.()]+)'
+          # Example: Cisco IOS Software, IOS-XE Software, Catalyst L3 Switch Software (CAT3K_CAA-UNIVERSALK9-M), Version 03.06.06E RELEASE SOFTWARE (fc1) Technical Support: http://www.cisco.com/techsupport Copyright (c) 1986-2016 by Cisco Systems, Inc. Compiled Sat 17-Dec-
+      model:
+        symbol:
+          OID: 1.3.6.1.2.1.1.1.0
+          name: sysDescr
+          extract_value: '\sSoftware\s+\(([-a-zA-Z0-9_]+)\)'
+          # Example: Cisco IOS Software, IOS-XE Software, Catalyst L3 Switch Software (CAT3K_CAA-UNIVERSALK9-M), Version 03.06.06E RELEASE SOFTWARE (fc1) Technical Support: http://www.cisco.com/techsupport Copyright (c) 1986-2016 by Cisco Systems, Inc. Compiled Sat 17-Dec-
+
```

Comment:
For `os_name` and `product_name` we to implement II-645 first to Support multiple symbols for the same field for profile metadata.